content/docs/handling-events.md

```diff
@@ -91,18 +91,18 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/xEmzGg?editors=0010)
+[**在 CodePen 上試試看吧！**](https://codepen.io/gaearon/pen/xEmzGg?editors=0010)
 
-You have to be careful about the meaning of `this` in JSX callbacks. In JavaScript, class methods are not [bound](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) by default. If you forget to bind `this.handleClick` and pass it to `onClick`, `this` will be `undefined` when the function is actually called.
+請特別注意 `this` 在 JSX callback 中的意義。在 JavaScript 中，class 的方法在預設上是沒有被綁定（[bound](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind)）的（譯註：也就是說 `handleClick` 內的 `this` 在預設上是沒有綁定任何物件的）。如果你忘了綁定 `this.handleClick` 並把它傳遞給 `onClick` 的話，`this` 的值將會在該 function 被呼叫時變成 `undefined`。
```

Comment:
```suggestion
請特別注意 `this` 在 JSX callback 中的意義。在 JavaScript 中，class 的方法在預設上是沒有被綁定（[bound](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Global_objects/Function/bind)）的。如果你忘了綁定 `this.handleClick` 並把它傳遞給 `onClick` 的話，`this` 的值將會在該 function 被呼叫時變成 `undefined`。
```

I think to keep original content, don't add more translator comment to explain something.